content/response-to-reviewers.md

```diff
@@ -84,35 +89,56 @@ Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/49>
 
 > p. 3, paragraph 2: research advisors don't seem to be the best proxy for senior faculty who would be invited for keynotes or honored as fellows
 
-Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/50>
+We did not make this point clear.
+What we meant was to contrast advisors with their trainees.
```

Comment:
```suggestion
What we meant was to contrast advisors with their trainees as the authors on a specific paper.
```

Reply:
I don't like this wording, but you had a point where you noted that what we were trying to do was ask: "who is in the type of position that would be keynote eligible _within_ the paper's authors" and I didn't think the current phrasing here conveyed that.